tests/main.js

```diff
@@ -506,6 +543,70 @@ testMany(
   },
 )
 
+testMany(
+  'Can bundle native ESM functions when the Node version is >=14 and the `zisi_pure_esm` flag is on',
+  ['bundler_nft'],
+  async (options, t) => {
+    const length = 2
+    const fixtureName = 'node-esm'
+    const opts = merge(options, {
+      basePath: `${FIXTURES_DIR}/${fixtureName}`,
+      featureFlags: { zisi_pure_esm: true },
+    })
+    const { files, tmpDir } = await zipFixture(t, fixtureName, {
+      length,
+      opts,
+    })
+
+    await unzipFiles(files, (path) => `${path}/../${basename(path)}_out`)
+
+    const functionPaths = [join(tmpDir, 'func1.zip_out', 'func1.js'), join(tmpDir, 'func2.zip_out', 'func2.js')]
+    const func1 = await import(functionPaths[0])
```

Comment:
`pathToFileURL()` must be used, like in https://github.com/netlify/zip-it-and-ship-it/pull/980/files#diff-91254b3a8e85bba2e8bad93724729fcd3531241f5b57196acc1e49db3e856a49R113

This is because `import()` argument is a URI, not a file path (unlike `require()`).
On Windows, absolute file paths like `C:\...` are not valid URLs. Also, URIs are percent-encoded, so this would also fail for file paths with characters which would normally be percent-encoded.

Reply:
Thank you! Done in https://github.com/netlify/zip-it-and-ship-it/pull/1004/commits/7156614214c1350c0174fce855119d17289f562b.